tests/support/zfs.py

```diff
@@ -197,9 +213,11 @@ def __init__(self):
                     "        groupquota@...  YES       NO   <size> | none",
                     "        written@<snap>   NO       NO   <size>",
                     "",
```

Comment:
We should probably not blacken this entire section... it makes it far less readable